frontend/src/scenes/insights/trendsLogic.js

```diff
@@ -323,6 +323,7 @@ export const trendsLogic = kea({
         },
         setFilters: async () => {
             insightLogic.actions.setAllFilters(values.filters)
+            actions.loadResults()
```

Comment:
after the changes you made, the filtesr and searchparams are somehow always synced up so the load results call needs to be done here instead of in the urltoaction logic.

sidenote: from what I can tell this works, but my local environment is being extremely buggy unresponsive (unrelated to this PR because i checked on master and it's equally buggy). lmk if your QA shows it working